specifications/M1/verifiable.presentation.protocol.md

```diff
@@ -171,7 +171,40 @@ the `credentials` endpoint
 
 `POST /credentials`
 
-If successful, credentials will be created or updated and a `200 OK` returned.
+If the POST is successful, credentials will be created and a `201 Created` returned.
+
+The POST body is a JSON object with the following properties:
+
+- `@context`: REQUIRED. Specifies a valid [Json-Ld context](https://www.w3.org/TR/json-ld11/#the-context).
+- `@type`: REQUIRED. A string specifying the credential type.
+- `credentials`: REQUIRED. A Json structure corresponding to the schema
+  specified [below](#the-credential-object).
+
+The following is a non-normative example of the JSON body:
+
+```json
+{
+  "@context": [
+    "https://w3id.org/tractusx-trust/v0.8"
```

Comment:
We should not tie this to Tractus-X
```suggestion
    "https://w3id.org/example-context/v0.0.1"
```

Reply:
Right now we need to. Eventually, this will get tied to the Eclipse WG namespace but for now, it has to be this way. Also, the versioning should stay at 0.8 to be consistent with DSP